src/components/OptionsSelector/BaseOptionsSelector.js

```diff
@@ -276,7 +276,6 @@ class BaseOptionsSelector extends Component {
                 value={this.props.value}
                 label={this.props.textInputLabel}
                 onChangeText={this.props.onChangeText}
-                placeholder={this.props.placeholderText}
```

Comment:
I think we can/should remove the references to `placeholderText` from [optionsSelectorPropTypes](https://github.com/Expensify/App/blob/93a7ecd7788f6b0ca5c465a4f7ccf49401e93ede/src/components/OptionsSelector/optionsSelectorPropTypes.js#L43) too now right?

Reply:
> However, it seems like there is one [component](https://github.com/allroundexperts/Expensify/blob/75323d117d10576360444ed53d95d841b97e9eb4/src/pages/settings/Profile/PronounsPage.js#L137-L138) that uses both placeholderText and textInputLabel.

Missed that!!!

> If @shawnborton agrees, we can remove placeholderText from this component as well. If we do this, we can completely remove the placeholderText prop.

I think it makes sense if we all want that! Let's wait on @shawnborton

Reply:
Seems like we should keep it since we're using it on the pronoun page right? Or maybe I am misunderstanding you.